pallets/liquidity-pools-gateway/src/message.rs

```diff
@@ -3,19 +3,19 @@ use frame_support::pallet_prelude::{Decode, Encode, MaxEncodedLen, TypeInfo};
 
 /// Message type used by the LP gateway.
 #[derive(Debug, Encode, Decode, Clone, Eq, MaxEncodedLen, PartialEq, TypeInfo)]
-pub enum GatewayMessage<AccountId, Message> {
+pub enum GatewayMessage<Message> {
 	Inbound {
 		domain_address: DomainAddress,
 		message: Message,
 	},
 	Outbound {
-		sender: AccountId,
+		sender: DomainAddress,
```

Comment:
Normally it will be `DomainAddress::Centrifuge(AccountId)`, but if in the future we want to relay some message we should be able to storage a different origin

Reply:
The forwarding wrap includes the source domain as well as the forwarding contract. But that needs to be part of the message such that EVM can distinguish between forwarded and normal messages.

Reply:
Makes sense